app/controllers/alchemy/pages_controller.rb

```diff
@@ -84,6 +85,21 @@ def sitemap
 
     private
 
+    # Redirects to requested action without locale prefixed
+    def enforce_no_locale
+      redirect_permanently_to additional_params.merge(locale: nil)
+    end
+
+    # Is the requested locale allowed?
+    #
+    # If Alchemy is not in multi language mode or the requested locale is the default locale,
+    # then we want to redirect to a non prefixed url.
+    #
+    def locale_prefix_not_allowed?
```

Comment:
Could this be rewritten to positive? 

```
def locale_prefix_allowed? 
  multi_language? && params[:locale] && params[:locale].presence != ::I18n.default_locale.to_s
end
```

Reply:
Although I just moved and did not introduced the method in this PR I like that idea.

Reply:
Tried to change it to the proposed method. It leads to infinite redirects. We can rename this method anytime. Will leave it as it was for now.